### DISCUSSION TITLE
zsh: no such file or directory: eval "$(/opt/homebrew/bin/brew shellenv)"

### DISCUSSION THREAD

**Discussion by FR86415** [Homebrew/discussions#3674] [Getting started]
I can't start. 

frankdr.ritzler@Franks-MacBook-Pro-2 ~ % 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> /Users/frankdr.ritzler/.zprofile    
    eval "$(/opt/homebrew/bin/brew shellenv)"
zsh: no such file or directory: eval "$(/opt/homebrew/bin/brew shellenv)"

**Comment by SMillerDev** [comment]
You forgot to copy the echo command

**Reply by FR86415** [reply]
Thank you
